docs/src/topics/firewalling.md

```diff
@@ -0,0 +1,75 @@
+# Firewalling
+
+This guide covers how Cilium can be set up to act as a [host firewall](https://docs.cilium.io/en/latest/security/host-firewall/) on CAPL clusters.
+
+## Default Configuration
+By default, the following policies are set to audit mode(without any enforcement) on CAPL clusters
+
+* [Kubeadm](./flavors/default.md) cluster allow rules
+
+    | Ports     | Use-case                 | Allowed clients       |
+    |-----------|--------------------------|-----------------------|
+    | 6443      | API Server Traffic       | World                 |
+    | 2379-2380 | Etcd Traffic             | World                 |
+    | *         | In Cluster Communication | Intra Cluster Traffic |
+
+
+* [k3s](./flavors/k3s.md) cluster allow rules
+    
+    | Ports | Use-case                 | Allowed clients               |
+    |-------|--------------------------|-------------------------------|
+    | 6443  | API Server Traffic       | World                         |
+    | *     | In Cluster Communication | Intra Cluster and VPC Traffic |
+
+* [RKE2](./flavors/rke2.md) cluster allow rules
+
+  | Ports | Use-case                 | Allowed clients               |
+  |-------|--------------------------|-------------------------------|
+  | 6443  | API Server Traffic       | World                         |
+  | 9345  | RKE2 Management Traffic  | All                           |
```

Comment:
dumb question but what's the difference between World vs All?

Reply:
no difference, I'll standardize on the world terminology